go.mod

```diff
@@ -114,3 +114,5 @@ require (
 )
 
 go 1.19
+
+replace github.com/codeready-toolchain/api => github.com/mfrancisc/api v0.0.0-20230814083608-6d8633a3bcf3
```

Comment:
Please update the relevant api repo first and then update thus replicate statement, instead.

Reply:
This is done only for testing (unit and e2e) the PR with the updated version of the API repo, once all the PRs are approved and passing all the tests/checks, we:
- merge the API pr
- update other PRs with the new version
- merge other repos with new master API version

we also have a CI check in place which prevents from merging if there is any `replace` statement, see: https://github.com/codeready-toolchain/host-operator/actions/runs/5857146355/job/15878356586?pr=848

So this will be replaced once all the tests are passing and all the PRs are approved.

Hopefully this answers you question/concern. But plz let me know if I misunderstood.